dali/test/python/test_operator_warp.py

```diff
@@ -163,15 +166,17 @@ def test_cpu_vs_cv():
   for batch_size in [1, 4, 19]:
     for use_input in [False, True]:
       for (itype, otype) in io_types:
+        inv_map = random.choice([False, True])
```

Comment:
just wondering, is it possible to fix the seed for deterministic results? Not sure if we want that or not. @mzient ?

Reply:
it's fixed. line 33:
`random.seed(1009)`

Reply:
That's good but not ideal - if there are multiple tests involving `random`, they would differ in results depending on which tests are run.

Reply:
also:
```suggestion
        inv_map = random.choice([False, True, None])
```
?